### DISCUSSION TITLE
Looks like I'm unable to use fflate together with brunch

### DISCUSSION THREAD

**Discussion by zzynx** [101arrowz/fflate#218] [Q&A]
I want to use fflate in a React/Typescript web app project.
I'm using Brunch (https://brunch.io/) as package manager.
In it's brunch-config.js file there is (among others):
```
    files : {
        javascripts: {
            joinTo: {
                'vendor.js': /^(?!src)/,
                'app.js': /^src\/main\/webapp\/app.*\.(ts|tsx)/
            }
        },
```
I added fflate in the dependencies of my package.json file (and ran yarn install) and imported it in one of my source files:
```
import { compressSync, strToU8 } from 'fflate';
```
However when I run my built web app it complains that it can't find the module fflate.
And indeed when I open my vendor.js file and search for 'fflate', it can't be found while I do find other libraries I use.
All libraries have their subdirectory in the directory node_modules, including fflate.

Any idea why fflate is not included in the vendor.js file brunch creates? 
(and hence unfortunately I can't use fflate in my project)

**Comment by 101arrowz** [comment]
This sounds similar to some issues that existed with Webpack in the past where files with `.cjs` file extensions were not treated as JavaScript files. (`.cjs` is just a JavaScript file with CommonJS modules.)

Could you check if Brunch supports the `.cjs` extension? Maybe you need to change your regex somehow to add the files in.
